docs/batch/batch_registration.md

```diff
@@ -0,0 +1,67 @@
+# Batch Registration
+
+To register a new batch, start by [navigating](batch_introduction.md#batch-navigation) 
+into the batch summary view.
+![batch_summary](images/batch_summary_no_batches.png)
+
+!!! info "Email Notification"
+    Upon successful batch registration, all project collaborators
+    will automatically receive an email with a link to the created batch.
+
+Within this view, click on the register button within the batch registration component on the top right to 
+trigger the batch registration dialog.
+![batch_registration_dialog](images/batch_registration_dialog.png)
+
+Within this dialog you have the possibility to provide the minimal required batch and sample information.
+
+The minimal batch information consists of:
+
+1. Batch name
+2. The [sample specific mandatory metadata information](#sample-registration) for at least one sample.
+
+## Sample Registration
+
+The sample specific mandatory metadata information hast to be defined within the spreadsheet of the dialog.
+Each row within this spreadsheet represents one sample within your batch. 
+You can add or remove rows via the dedicated control buttons on top right of the spreadsheet. 
+For a high amount of samples, we recommend to click on the "Prefill spreadsheet" button 
+to trigger an automatic prefilling of the spreadsheet with the information provided during [experiment creation](experiment_creation.md)
+
+!!! tip "Prefill Sample Information"
+    The "Prefill Spreadsheet" functionality will generate one row according to the number of biological replicates and condition defined during
+    [experimental group creation](experiment_creation.md#experimental-group-creation).
+
+Mandatory metadata properties can be recognized via the asterisk next to the column header and consist of:
+
+1. Analysis to be performed 
+2. Sample Label
+3. Condition 
+4. Species
+5. Specimen
+6. Analyte
+
+!!! note "Preselected Information"
+    Cells within the "Species", "Specimen" and "Analyte" columns provide the values specified during [experiment creation](experiment_creation.md).
+    Cells within the "Condition" column provide a selection of the values specified during [experimental group creation](experiment_creation.md#experimental-group-creation)
+
+Optionally, feel free to also store a comment of your choosing and the ontology specific [Organism ID](ontology#ontology-search) metadata properties for each sample.
```

Comment:
Careful, this is wrong information. The Organism ID is to distinguish different patients, animals or plants the sample was extracted from. For example one could set "patient 1", "patient 2" etc.